Content.Client/Overlays/EquipmentHudSystem.cs

```diff
@@ -104,7 +104,9 @@ protected virtual void OnRefreshComponentHud(EntityUid uid, T component, Refresh
 
     private void RefreshOverlay(EntityUid uid)
     {
-        if (uid != _player.LocalSession?.AttachedEntity)
+        if (_player.LocalSession?.AttachedEntity != null)
+            uid = _player.LocalSession.AttachedEntity.Value;
+        else
             return;
```

Comment:
Should the uid even be passed in at this point? Seems like it should just be removed in lieu of always doing local player.

Reply:
Yeah this is a really old PR but there never was a reason to pass the uid because it just returns if it doesn't match the local player